### PR TITLE
Updated the json schema and the deprecated draft7 format checker

### PR DESCRIPTION
**Pull request by chandrams**
Updated the json schema and the deprecated draft7 format checker

### CODE REVIEW

File: tests/scripts/common/common_functions.sh [hunk 1]
```diff
@@ -277,7 +277,8 @@ function autotune_cleanup() {
 	echo "CMD = ${cmd}"
 	${cmd} >> ${AUTOTUNE_SETUP_LOG} 2>&1
 	# Remove the prometheus setup
-	prometheus_cleanup
+	# Uncomment this line if prometheus setup also needs to be cleaned up
+	# prometheus_cleanup
```
Comment (dinogun) [inline tests/scripts/common/common_functions.sh:281]: Maybe this needs to be a parameter? Especially because this is common code

Comment (chandrams) [reply tests/scripts/common/common_functions.sh]: Thanks Dino, I have updated the scripts with an option to cleanup prometheus.

**Comment by chandrams** [comment]
@dinogun - Can you please review this PR, thanks.

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**